### DISCUSSION TITLE
Remove content tags

### DISCUSSION THREAD

**Discussion by bershanskiy** [mdn/mdn-community#123] [Content]
Prior to [MDN redesign](https://hacks.mozilla.org/2022/03/a-new-year-a-new-mdn/) launched in March, MDN pages used to display tags. These tags are no longer used, but the underlying data lives on in YAML headers in [mdn/content](https://github.com/mdn/content). These tags were never particularly useful since their use was rather inconsistent. Since these tags are no longer used, I suggest we remove them altogether.

**Comment by teoli2003** [comment]
Removing the `Tags:` YAML header is indeed our medium-term goal. (It was a long-term goal back in 2018, and progress was made towards it, yeah!).

A few comments:

1. Tags were an attempt in the wiki era to store metadata on a page itself.
2. The tags weren't displayed on the site long before March 2022. I would say at least since the previous redesign, back in 2018 (I've not double-checked the year).
3. It is incorrect that they are not used: sidebars rely on them, for example, both for the page type (Interface, Property, Method, Constructor, …) and the feature status (experimental, deprecated, …)
4. We want to get rid of them for a long time. Before doing this, we must remove all uses of them. The `page-type` project is one step in this direction (as was the migration to Git, with pages using YAML and MarkDown), as it encodes some of the metadata currently stored in tags. The `page-type` YAML header will replace _Interface_, _Method_, _Property_, and a few others. Once the page type project is over, quite a few macros will need to be adapted.

Theoretically, we could remove some of them now, but I'm not sure it is useful. Once all uses of tags have been removed, it will be trivial to delete the whole header with a script.
